docs/products/flink/overview.rst

```diff
@@ -0,0 +1,97 @@
+Service overview
+================
+
+This article gives you an overview of the service plans and features available in Aiven for Apache Flink beta.
+
+You can run Aiven for Apache Flink on AWS or GCP cloud environments. Various deployment options are available, ranging from fully managed access over the public internet to subnet deployments under your own cloud account. All services and infrastructure are single-tenant and dedicated solely to a single customer.
+
+Aiven offers single-node and three-node Apache Flink clusters:
+
+* Single-node clusters are not highly available, but support automatic failover. They start from 4GB RAM/2 vCPU and extend up to 32GB/8 vCPU. This setup is a good option for proof of concepts and preliminary testing for development purposes.
+
+* Three-node clusters are highly available, support automatic failover, and range from 4GB RAM/2 vCPU to 32GB/8 vCPU.
+
+Our product page provides you detailed information about the available plans and pricing. 
+
+
+Managed service features
+------------------------
+
+**Default deployment configuration**
+  Aiven for Apache Flink is configured to use the `HashMap state backend <https://ci.apache.org/projects/flink/flink-docs-stable/api/java/org/apache/flink/runtime/state/hashmap/HashMapStateBackend.html>`_. This means that the amount of `state <https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/concepts/stateful-stream-processing/#what-is-state>`_ kept depends on the amount of memory available in the cluster. 
+
+  The Flink cluster executes applications in `session mode <https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/deployment/overview/#session-mode>`_, so you can deploy multiple Flink jobs on the same cluster to use the available resources effectively.
+
+**Scaling a Flink cluster**
+  Each node is equipped with a TaskManager and JobManager. We recommend that you scale up your cluster to add more CPU and memory for the TaskManager before attempting to scale out. This approach makes the best use of the resources available in the nodes.
+
+  By default, each created TaskManager is configured with a single slot for best isolation between jobs. We highly recommend that you adjust this setting for your use case.
```

Comment:
Is there a resource where I can understand more about this option and why I should care?

Reply:
Maybe we can say something about a slot being = to 1 data pipeline

Reply:
Nothing separate as of yet, but I added the equivalency for reference and the setting path.